src/test/java/com/rackspace/ceres/app/utils/DateTimeUtilsTest.java

```diff
@@ -29,57 +27,59 @@ public void getAbsoluteTimeFromRelativeTimeTest_Invalid() {
 
   @Test
   public void isValidInstantInstanceTest() {
-    assertTrue(DateTimeUtils.isValidInstantInstance("2020-11-10T14:24:35Z"));
+    assertThat(DateTimeUtils.isValidInstantInstance("2020-11-10T14:24:35Z")).isTrue();
   }
 
   @Test
   public void isValidInstantInstanceTest_Invalid() {
-    assertFalse(DateTimeUtils.isValidInstantInstance("13:03:15.454+0530Z"));
+    assertThat(DateTimeUtils.isValidInstantInstance("13:03:15.454+0530Z")).isFalse();
   }
 
   @Test
   public void isValidEpochMillisTest() {
-    assertTrue(DateTimeUtils.isValidEpochMillis("1605094715000"));
+    assertThat(DateTimeUtils.isValidEpochMillis("1605094715000")).isTrue();
   }
 
   @Test
   public void isValidEpochMillisTest_Invalid() {
-    assertFalse(DateTimeUtils.isValidEpochMillis("1605094715"));
+    assertThat(DateTimeUtils.isValidEpochMillis("1605094715")).isFalse();
   }
 
   @Test
   public void isValidEpochSecondsTest() {
-    assertTrue(DateTimeUtils.isValidEpochSeconds("1605094715"));
+    assertThat(DateTimeUtils.isValidEpochSeconds("1605094715")).isTrue();
   }
 
   @Test
   public void isValidEpochSecondsTest_Invalid() {
-    assertFalse(DateTimeUtils.isValidEpochSeconds("1605094715000"));
+    assertThat(DateTimeUtils.isValidEpochSeconds("1605094715000")).isFalse();
   }
 
   @Test
   public void parseInstantTestWithNull() {
-    assertNotNull(DateTimeUtils.parseInstant(null));
+    assertThat(DateTimeUtils.parseInstant(null)).isNotNull();
   }
```

Comment:
Can you also assert it's within a second of current time, like you did in `getAbsoluteTimeFromRelativeTimeTest`

Reply:
Sorry...Missed that one